lib/ui/members/members_screen.dart

```diff
@@ -98,6 +98,19 @@ class _MembersScreen extends State<MembersScreen> {
                             colorFilter: greyScaleColorFilter,
                           ),
                         )),
+                    if (member.imageUrl == null) Container(
+                        margin: EdgeInsets.fromLTRB(0, 0, 10, 10),
+                        child: CircleAvatar(
+                          radius: 55,
+                          backgroundColor: primaryColorTeal,
+                          child: ColorFiltered(
+                            child: CircleAvatar(
+                              radius: 50,
+                              backgroundImage: AssetImage('assets/cincinnati_skyline.jpg'),
```

Comment:
@sierraobryan I couldn't figure out how to use a ternary of if/else statement on backgroundImage directly

Reply:
You should be able to do something like `backgroundImage: (url == null) ? AssesImage : NetworkImage` and not need to repeat any code! See if that helps (:

Reply:
Yup, that made it a lot more clean and concise! :)